__typetests__/react-native-reanimated-tests.tsx

```diff
@@ -57,10 +57,9 @@ import Animated, {
   StyleLayoutAnimation,
   Animation,
   // eslint-disable-next-line import/no-unresolved
-} from 'react-native-reanimated';
+} from '../src';
```

Comment:
I'm not a huge fan of this change, `import { ... } from 'react-native-reanimated';` also checks if there's a module called `react-native-reanimated`. Why is it necessary?

Reply:
In my environment TypeScript couldn't resolve this import properly and inferred all of the imports as `any`, making the tests kind-of useless. I'd be more than happy if you could check if that's the case for you too.

While this file can check if there's a module called `react-native-reanimated`, this file isn't considered anywhere in our CIs etc., hence even if there was something wrong there we wouldn't know until we've had it manually checked. I think this file should focus on types only - and if we want, we could do much more reliable test that checks if module called `react-native-reanimated` exists.